.gitignore

```diff
@@ -6,8 +6,6 @@
 
 target
 generated
-#Config file
-Config.toml
```

Comment:
Better to keep the `Config.toml` files in .gitignore, as there's a chance to mistakenly commit sensitive data otherwise.

Reply:
Fixed with [240c2ee4c7a77fa486c9219617f8c49cd9f2283e](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/13/commits/240c2ee4c7a77fa486c9219617f8c49cd9f2283e)